### PR TITLE
#185027364 Create Landing Page of the app

### PR DESCRIPTION
**Pull request by khevin2**
### What does this PR do?
This PR add `Homepage` to the application.
### Description of Task to be completed?

- Add Navigation
- Add hero
- Create Footer

### How should this be manually tested?

- Clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- Switch your working directory `cd e-comm-team-amigos-fn`
- Install all the dependencies by running `npm install`
- Then run the project by typing `npm start`

### Any background context you want to provide?

- Have Node Js installed in your local machine
- Have latest web browser

### What are the relevant pivotal tracker?

Story ID:  [#185027364](https://www.pivotaltracker.com/story/show/185027364)

### Screenshots

Screenshot from 2023-04-27 14-37-01

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "jsx": "react",
+        "strictNullChecks": true,
+        "strictFunctionTypes": true
+    },
+    "exclude": [
+        "node_modules",
+        "**/node_modules/*",
+        "dist"
+    ]
+}
```
Comment (Emile-SHUMBUSHO) [inline jsconfig.json:15]: Avoid an empty line from your code

File: src/components/Button/Button.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    border-radius: 10.16px;
+}
+
+oval80 {
+    border-radius: 80px;
+}
+
+oval50{
+    border-radius: 50px;
+}
```
Comment (Emile-SHUMBUSHO) [inline src/components/Button/Button.css:15]: Avoid an empty line from your code

File: src/components/Hero/Hero.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+height: 24px;
+background: #FFFFFF;
+border-radius: 50px;
+}
+
+.swiper-pagination-bullet-active {
+    width: 50px;
+height: 24px;
+background: #163C2A;
+}
```
Comment (Emile-SHUMBUSHO) [inline src/components/Hero/Hero.css:46]: Avoid an empty line from your code

File: src/components/TodayDeal/style.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    font-weight: 300;
+    font-size: 12px;
+    line-height: 18px;
+    /* identical to box height */
+
+    display: flex;
+    align-items: center;
+
+    color: #000000;
+}
```
Comment (Emile-SHUMBUSHO) [inline src/components/TodayDeal/style.css:77]: Avoid an empty line from your code

File: src/components/TopCategories/style.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+}
+.swiper-button-next:after {
+        font-family: swiper-icons;
+    font-size: 700 !important;
+    text-transform: none !important;
+    letter-spacing: 0;
+    font-variant: initial;
+    line-height: 1;
+    color: #096F3E;
+}
```
Comment (Emile-SHUMBUSHO) [inline src/components/TopCategories/style.css:19]: Avoid empty line from your code

Comment (khevin2) [reply src/components/TopCategories/style.css:19]: What do you mean by 'empty lines'?

**Review by GasanaJr — CHANGES_REQUESTED** [review]
Amazing Job, Khevin 🫡. However rebase and fix tests

**Review by wilbrord2 — CHANGES_REQUESTED** [review]
work on the slider, the responsiveness on big screen, and add some padding on some sections

**Review by GasanaJr — CHANGES_REQUESTED** [review]
The tests still have an issue bro

File: src/components/Hero/index.jsx [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+import colors from '../../constants/colors';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Button from '../Button';
+
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Autoplay } from 'swiper';
+import 'swiper/css';
+
```
Comment (GasanaJr) [inline src/components/Hero/index.jsx:11]: Shouldn't it be `swiper.css` instead of `swiper/css`? Because the tests are failing because of this

Comment (khevin2) [reply src/components/Hero/index.jsx:11]: It's actually `swiper/css`. and I'm searching for a way the tests could pass using this method

**Comment by Emile-SHUMBUSHO** [comment]
Put some margin on cart, otherwise LGTM

Screenshot 2023-05-04 at 08 42 58

**Review by AbdulKhaliq59 — APPROVED** [review]
it LGTM

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]

**Pull request merged**